src/com/company/Main.java

```diff
@@ -1,75 +1,84 @@
 package com.company;
 
+import com.dbconnectors.DbConnector;
+
 import java.io.File;
-import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.StringTokenizer;
+import java.sql.SQLException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 
 public class Main {
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String SPECIAL_CHAR_REMOVAL_REGEX = "[^a-zA-Z0-9@]";
-    final static String SINGLE_SPACE = " ";
 
-    public static void main(String[] args) {
+    static int count = 0;
+    private static DbConnector database = new DbConnector();
 
+    public static void main(String[] args) throws SQLException, InterruptedException {
+        String txtFilePath = null;
+        String searchWord = null;
         if (args.length != 2) {
-            System.out.println("Exiting, since no file path and word to search for are provided!");
+            System.out.println(ErrorMessage.ERROR_MESSAGE_PARAMETERS_NOT_FOUND);
+            database.databaseConnector(txtFilePath, searchWord, count, Constants.STATUS_FAILURE, ErrorMessage.ERROR_MESSAGE_PARAMETERS_NOT_FOUND);
+            return;
+        }
+        txtFilePath = args[0];
+        searchWord = args[1];
+        if (searchWord == null || searchWord.trim().isEmpty()) {
+            System.out.println(ErrorMessage.ERROR_MESSAGE_WORD_IS_NULL);
+            database.databaseConnector(txtFilePath, searchWord, count, Constants.STATUS_FAILURE, ErrorMessage.ERROR_MESSAGE_WORD_IS_NULL);
+            return;
+        }
+        if (txtFilePath == null || txtFilePath.equals(" ") || txtFilePath.equals("")) {
```

Comment:
The same logic in line 28 should be used here, txtFilePath can be any number of empty spaces.

Or you can just call txtFilePath.isBlank()

Reply:
replaced the code by using the isBlank() method.